src/version.h

```diff
@@ -12,9 +12,26 @@
 static const int PROTOCOL_VERSION = 170002;
 
 //! initial proto version, to be increased after version/verack negotiation
-static const int INIT_PROTO_VERSION = 170002;
+static const int INIT_PROTO_VERSION = 209;
```

Comment:
I just tried reverting solely this change, and I still encounter degraded network behaviour. So ACK on reverting the entire PR.